beta/README.md

```diff
@@ -1,5 +1,122 @@
-# Beta Deployments
+# Beta deployments
 
 This folder contains beta versions of 1Password SCIM bridge deployments.
 
-These deployments *should* work, but come with no guarantees, and will change in the future.
+These deployments _should_ work, but come with no guarantees, and will change in the future.
+
+> **Note**  
+> "Beta" refers solely to the deployment _method_. It does not refer to the SCIM bridge software being deployed.
+
+## What are "beta" deployment examples?
+
+We want to provide people deploying 1Password SCIM Bridge with deployment examples suitable for a range of skill levels, environments, and budgets. As part of those efforts, we may choose to make deployment examples available in "beta" form. This gives people deploying 1Password SCIM Bridge more options and provides us with feedback we can use to enhance the deployment example.
+
+### What does "beta" mean?
+
+"Beta" refers to the deployment example. It does not refer to the software being deployed, which in this case is 1Password SCIM Bridge. You will always be deploying a stable version of 1Password SCIM Bridge, regardless of how you've deployed it.
+
+Generally speaking, any new deployment example will start in beta. This allows us better-assesss where documentation could be enhanced, where the deployment could be improved, identify edge-cases we need to accommodate, and so on.
```

Comment:
allows us to better assess

Reply:
I don't think 4 s's in assess is enough, I like the fifth one :P

Reply:
```suggestion
Generally speaking, any new deployment example will start in beta. This allows us better assess where documentation could be enhanced, where the deployment could be improved, identify edge-cases we need to accommodate, and so on.
```